decidim-conferences/lib/decidim/conferences/admin_engine.rb

```diff
@@ -97,113 +97,131 @@ class AdminEngine < ::Rails::Engine
+                        active: is_active_link?(decidim_admin_conferences.edit_conference_path(current_participatory_space))
+          menu.add_item :components,
+                        I18n.t("components", scope: "decidim.admin.menu.conferences_submenu"),
+                        decidim_admin_conferences.components_path(current_participatory_space),
+                        active: is_active_link?(decidim_admin_conferences.components_path(current_participatory_space)),
+                        if: allowed_to?(:read, :component, conference: current_participatory_space),
+                        submenu: { target_menu: :admin_conferences_components_menu, options: { container_options: { id: "components-list" } } }
+
+          menu.add_item :categories,
+                        I18n.t("categories", scope: "decidim.admin.menu.conferences_submenu"),
+                        decidim_admin_conferences.categories_path(current_participatory_space),
+                        active: is_active_link?(decidim_admin_conferences.categories_path(current_participatory_space)),
+                        if: allowed_to?(:read, :category, conference: current_participatory_space)
+
+          menu.add_item :attachments,
+                        I18n.t("attachments", scope: "decidim.admin.menu.conferences_submenu"),
+                        "#",
+                        active: is_active_link?(decidim_admin_conferences.conference_attachment_collections_path(current_participatory_space)) ||
+                                is_active_link?(decidim_admin_conferences.conference_attachments_path(current_participatory_space)),
+                        if: allowed_to?(:read, :attachment_collection, conference: current_participatory_space) ||
+                            allowed_to?(:read, :attachment, conference: current_participatory_space),
+                        submenu: { target_menu: :conferences_admin_attachments_menu }
+          menu.add_item :conference_media_links,
+                        I18n.t("media_links", scope: "decidim.admin.menu.conferences_submenu"),
+                        decidim_admin_conferences.conference_media_links_path(current_participatory_space),
+                        if: allowed_to?(:read, :media_link, conference: current_participatory_space),
+                        active: is_active_link?(decidim_admin_conferences.conference_media_links_path(current_participatory_space))
+          menu.add_item :conference_partners,
+                        I18n.t("partners", scope: "decidim.admin.menu.conferences_submenu"),
+                        decidim_admin_conferences.conference_partners_path(current_participatory_space),
+                        if: allowed_to?(:read, :partner, conference: current_participatory_space),
+                        active: is_active_link?(decidim_admin_conferences.conference_partners_path(current_participatory_space))
+          menu.add_item :conference_speakers,
```

Comment:
```suggestion

          menu.add_item :conference_speakers,
```

Reply:
my understanding is that you requested to add a new line.